torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1519,6 +1519,56 @@ at::Tensor leaky_relu_backward(const at::Tensor& grad_output,
       negative_slope.to<double>()));
 }
 
+at::Tensor lerp(const at::Tensor& self, const at::Tensor& end,
+                const at::Tensor& weight) {
+  XLA_FN_COUNTER("xla::");
+  return bridge::AtenFromXlaTensor(
+      XLATensor::lerp(bridge::GetXlaTensor(self), bridge::GetXlaTensor(end),
+                      bridge::GetXlaTensor(weight)));
+}
+
+at::Tensor lerp(const at::Tensor& self, const at::Tensor& end,
+                const at::Scalar& weight) {
+  XLA_FN_COUNTER("xla::");
+  return bridge::AtenFromXlaTensor(XLATensor::lerp(
+      bridge::GetXlaTensor(self), bridge::GetXlaTensor(end), weight));
+}
+
+at::Tensor& lerp_(at::Tensor& self, const at::Tensor& end,
+                  const at::Tensor& weight) {
+  XLA_FN_COUNTER("xla::");
+  XLATensor self_tensor = bridge::GetXlaTensor(self);
+  XLATensor::lerp_(self_tensor, bridge::GetXlaTensor(end),
```

Comment:
If we delete the inplace/out= kernels and let the code-generation take care of it, these tests will need to be updated to call the code-generated kernels, like in the refactor from [here](https://github.com/pytorch/xla/pull/2962/files#diff-35e95d49f658a3fbe4a3ec5cc0fc90fc969366a68caf065fb805ab93235aed9fR209)

Reply:
I think I already use `tensor.lerp_` like you did in `test_tensor.cpp`. Could you take a look and let me know if there is other things I need to change?

Reply:
yep, looks fine!